### DISCUSSION TITLE
I manually deleted tables then the public schema in a PostgreSQL database

### DISCUSSION THREAD

**Discussion by SomeUserNameReally** [prisma/prisma#7659] [Q&A] [answered]
After deleting them manually, running `npx prisma migrate dev --name init` doesn't add them back.

How can I restore the schema and the tables through Prisma?

 I tried looking through the documentation, but I wasn't able to find what would help me.

**Comment by SomeUserNameReally** [comment, marked as answer]
It was an issue with `.env` file.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
